Gemfile

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '2.6.3'
+gem 'bootsnap', '>= 1.1.0', require: false
+gem 'bootstrap', '~> 4.3.1'
+gem 'cancancan', '~> 3.0.1'
+gem 'coffee-rails', '~> 4.2'
+gem 'devise', '~> 4.6', '>= 4.6.2'
+gem 'jbuilder', '~> 2.5'
+gem 'kaminari'
+gem 'mini_racer', platforms: :ruby
+gem 'pg'
+gem 'puma', '~> 3.11'
+gem 'rails', '~> 5.2.3'
+gem 'sass-rails', '~> 5.0'
+gem 'slim', '~> 4.0.1'
+gem 'uglifier', '>= 1.3.0'
+gem 'webpacker', '~> 4.x'
+
+group :development, :test do
+  gem 'byebug', platforms: %i[mri mingw x64_mingw]
```

Comment:
For me, pry is better. Look into `pry-byebug` gem or `pry-rails`.

Reply:
I think that pry is better too. Guys also use byebug. I think that it is acceptable to keep both for now